pkgs/by-name/sw/sword/package.nix

```diff
@@ -59,6 +91,5 @@ stdenv.mkDerivation (finalAttrs: {
     '';
     license = lib.licenses.gpl2;
     maintainers = with lib.maintainers; [ AndersonTorres greg ];
-    platforms = lib.platforms.unix;
```

Comment:
Instead of removing it try setting it to `lib.platforms.all`

Reply:
Fixed, along with nixfmt errors

Reply:
`all` is not a synonym of `unix ++ windows`.

Reply:
Sword attempts to support building on every platform for which there is a C compiler available. If there's one it's still not succeeding on, I'd like to look into it before I set it as not supported on that platform.